app/middlewares/authCheck.js

```diff
@@ -11,3 +12,18 @@ exports.checkUser = (req, res, next) => {
     next(errors.unauthorizedNoLogin);
   }
 };
+
+exports.checkValidUserId = (req, res, next) => {
+  return User.findOne({ where: { id: req.params.user_id } })
+    .then(user => {
+      if (user) {
```

Comment:
you should also check that the id requested is the same of the id in the jwt, otherwise any user could see the albums that somebody else bought.